ERP/app/Http/Controllers/LogController.php

```diff
@@ -52,51 +52,43 @@ public function exportLogs(Request $request){
 
     }
 
-    //functions to create PDF
-    function get_logs()
-    {
-     $logs = DB::table('logs')
-         ->limit(10)
-         ->get();
-     return $logs;
-    }
-
-    function pdf()
-    {
-     $pdf = \App::make('dompdf.wrapper');
-     $pdf->loadHTML($this->convert_logs_to_html());
-     return $pdf->stream();
-    }
-
+    //function to convert logs to html
     function convert_logs_to_html()
     {
-     $logs = $this->get_logs();
-     $output = '
-     <h3 align="center">logs</h3>
-     <table width="100%" style="border-collapse: collapse; border: 0px;">
-      <tr>
-      <th style="border: 1px solid; padding:12px;" width="10%">Type</th>
-      <th style="border: 1px solid; padding:12px;" width="20%">TimeStamp</th>
-      <th style="border: 1px solid; padding:12px;" width="20%">IP Address</th>
-      <th style="border: 1px solid; padding:12px;" width="20%">Message</th>
-      <th style="border: 1px solid; padding:12px;" width="20%">Request Type</th>
-      </tr>
-      ';  
-
-     foreach($logs as $log)
-        {
-        $output .= '
+        $logs = Log::all()->sortByDesc('created_at');
```

Comment:
Could you comment this part? Ie. What it does

Reply:
More comments added